content/patterns/disclosure/disclosure-pattern.html

```diff
@@ -14,48 +14,49 @@
   </head>
   <body>
     <main>
-      <h1>Disclosure (Show/Hide) Pattern</h1>
+      <h1>ディスクロージャー（表示/非表示）パターン</h1>
+      <!-- af8f4c1d97ead402022d704754c3d382c4a2cef2 -->
 
       <section id="about">
-        <h2>About This Pattern</h2>
+        <h2>このパターンについて</h2>
         <p>
-          A disclosure is a widget that enables content to be either collapsed (hidden) or expanded (visible).
-          It has two elements: a disclosure <a href="../button/button-pattern.html">button</a> and a section of content whose visibility is controlled by the button.
-          When the controlled content is hidden, the button is often styled as a typical push button with a right-pointing arrow or triangle to hint that activating the button will display additional content.
-          When the content is visible, the arrow or triangle typically points down.
+          ディスクロージャーは、コンテンツを折りたたんだり（非表示）、展開したり（表示）できるウィジェットです。
+          ディスクロージャー<a href="../button/button-pattern.html">ボタン</a>と、ボタンによって表示/非表示が制御されるコンテンツのセクションの 2 つの要素があります。
```

Comment:
theを訳出した方がわかりやすいでしょう
```suggestion
          ディスクロージャー<a href="../button/button-pattern.html">ボタン</a>と、そのボタンによって表示/非表示が制御されるコンテンツのセクションの 2 つの要素があります。
```